ai8x.py

```diff
@@ -240,8 +240,7 @@ def __init__(self, min_val=None, max_val=None):
 
     def forward(self, x):  # pylint: disable=arguments-differ
         """Forward prop"""
-        x = x.clamp(min=self.min_val)
-        return x.clamp(max=self.max_val)
+        return x.clamp(min=self.min_val, max=self.max_val)
```

Comment:
@rotx-maxim Could you check this operation? Clamping was split into two parts in an [earlier commit](https://github.com/MaximIntegratedAI/ai8x-training/commit/8ce4fe52769a91606196184ee6a8e827cbd039da), and I'm not sure if reverting this change will break something.

Reply:
Splitting the clamp operation was due to an incompatibility with PyTorch 1.12. this may or may not be fixed in the latest 2.0 beta. Is there a compelling need to merge the operations (other than "it looks better")?

Reply:
There is no need to do this. Just to make the code better looking. I remember we did the change for one reason but could not remember the reason... I will undo this change!